apps/platform-app/src/app/[locale]/(profile)/(routes)/profiles/page.tsx

```diff
@@ -9,7 +9,7 @@ export default async function Profiles({
   searchParams: Record<JobOfferFiltersEnum, string>
 }) {
```

Comment:
**nitpick**: Self-closing tags can be used for the `Loader` component for cleaner syntax.